packages/abi-typegen/package.json

```diff
@@ -44,6 +44,8 @@
   ],
   "scripts": {
     "pretest": "run-s build:forc-callpaths build:forc",
+    "test": "cd ../.. && pnpm run test:filter packages/abi-typegen",
+    "test:update-fixtures": "UPDATE_FIXTURES=yes pnpm run test",
```

Comment:
Nit but can we use boolean over yes/no.

Reply:
Booleans are not very good for env values because they come always as strings.

```ts
true === Boolean(process.env.UPDATE_FIXTURES);
```

What do you think?

Reply:
I think that's better.

Reply:
Sorry, that what?

```ts
'yes' === process.env.UPDATE_FIXTURES;
true === Boolean(process.env.UPDATE_FIXTURES);
```

Reply:
Sorry I misinterpreted the above.

```ts
'true' === process.env.UPDATE_FIXTURES
```

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2824/commits/3a23bbdb861a02432b11409babeffcc02c6948f3